src/hooks/useCart/useCart.tsx

```diff
@@ -222,35 +230,16 @@ export const useCart = (
   );
 
   /**
-   * Handles the checking out of the cart.
-   * Sets checkoutResult to the response of the post transaction.
+   * POST the transaction and set the cartState according to completion
+   * or error caught.
    */
-  const checkoutCart: CartHook["checkoutCart"] = useCallback(() => {
-    const checkout = async (): Promise<void> => {
-      setCartState("CHECKING_OUT");
-
-      const allIdentifierInputs: IdentifierInput[] = [];
+  const closeCheckoutCart = useCallback(() => {
```

Comment:
Hmm "close" is a little strange in the context. Perhaps we can keep it simple with `_completeCheckout`? The underscore indicates it's an internal function